util/python/python_config.sh

```diff
@@ -46,7 +46,7 @@ function main {
 }
 
 function python_path {
-  python - <<END
+  $PYTHON_BIN_PATH - <<END
```

Comment:
In my system, $PYTHON_BIN_PATH seems to be undefined.
Are you sure this will work in all systems?